OSDOCS#8969: Adding descheduler 5.0.0 release notes (#70785)
openshift/openshift-docs: nodes/scheduling/descheduler/nodes-descheduler-release-notes.adoc

These release notes track the development of the {descheduler-operator}.

For more information, see xref:../../../nodes/scheduling/descheduler/index.adoc#nodes-descheduler-about_nodes-descheduler-about[About the descheduler].

[id="descheduler-operator-release-notes-5.0.0"]
== Release notes for {descheduler-operator} 5.0.0

Issued: 2024-02-28

The following advisory is available for the {descheduler-operator} 5.0.0:

* link:https://access.redhat.com/errata/RHEA-2024:0302[RHEA-2024:0302]

> [id="descheduler-operator-5.0.0-new-features-and-enhancements"]

Reviewer: ```suggestion

[id="descheduler-operator-5.0.0-new-features-and-enhancements"]
```

=== New features and enhancements

> * With this release, the {descheduler-operator} releases updates independently from the {product-title} minor version release stream.

Reviewer: ```suggestion
* With this release, the {descheduler-operator} releases updates independent of the {product-title} minor version release stream.
```

> [id="descheduler-operator-5.0.0-bug-fixes"]

Reviewer: ```suggestion

[id="descheduler-operator-5.0.0-bug-fixes"]
```

=== Bug fixes

> * Previously, the descheduler pod logs showed the following warning about the Operator's version: `failed to convert Descheduler minor version to float`. With this update, the warning is no longer shown. (link:https://issues.redhat.com/browse/OCPBUGS-14042[*OCPBUGS-14042*])

Reviewer: ```suggestion
* Previously, the descheduler pod logs showed the following warning about the Operator's version: 

[source,text]
----
`failed to convert Descheduler minor version to float`. 
----

With this update, the warning is no longer shown. (link:https://issues.redhat.com/browse/OCPBUGS-14042[*OCPBUGS-14042*])
```
